src/main/java/org/geppetto/datasources/ExecuteMultipleQueriesVisitor.java

```diff
@@ -120,17 +120,9 @@ public QueryResults getResults() throws GeppettoDataSourceException
 			boolean first = true;
 			for(QueryResults result : results.keySet())
 			{
-				if(finalResults.getHeader().isEmpty())
```

Comment:
@jrmartin why two PRs target the same piece of code? Are the both based on development or there is a sequence here? I foresee conflicts